app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/datamodel/Category.kt

```diff
@@ -0,0 +1,11 @@
+package com.github.HumanLearning2021.HumanLearningApp.datamodel
+
+/** Plain object for the database entity that represents an abstract category that learners can
+ * learn to recognize and classify.
+ */
+data class Category(
+    /** Name of the category in user language. */
+    val name: String? = null,
+    /** Level 1: presentation, (default) level 2: representation */
+    val comWoorLevel: Int = 1,
```

Comment:
it's written ComVoor. http://www.susa.be/index.php/services/formations/notre-catalogue/communiquer/142-sensibilisation-a-l-outil-comvoor
Also: wouldn't an enum be better here ?

Reply:
you could make some sort of adapter function in the enum maybe. but anyway, it's not gonna be an attribute of Category